src/navigations/MainNavigator.js

```diff
@@ -111,7 +111,7 @@ class MainNavigator extends React.Component {
 		if (
 			appState === 'background' &&
 			nextAppState === 'active' &&
-			restrictions.includes('app-opening')
+			restrictions.includes('APP_OPENING')
```

Comment:
Justly, it is not what i meant x)

`APP_OPENING` directly, not the string value ^^